sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -517,18 +517,25 @@ inline static void piQueueRetainNoLock(pi_queue Queue) { Queue->RefCount++; }
 // \param Queue pi_queue to associate with a new event.
 // \param Event a pointer to hold the newly created pi_event
 // \param CommandType various command type determined by the caller
-// \param ZeCommandList the handle to associate with the newly created event
+// \param CommandList is the command list where the event is added
 inline static pi_result
 createEventAndAssociateQueue(pi_queue Queue, pi_event *Event,
                              pi_command_type CommandType,
-                             ze_command_list_handle_t ZeCommandList) {
+                             pi_command_list_ptr_t CommandList) {
```

Comment:
This can be renamed CommandListMapIterator. And in the comment, we can mention we add the event to CommandListMapIterator->second

Reply:
I'd like to stay short in name and not have "Map"/"Iterator" in the many-many uses of command-lists